src/__tests__/fixtures/json-schema/vacationRentals.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
@himank can you also check this JSON payload to server for defaults

Reply:
looks good